man/org.freedesktop.systemd1.xml

```diff
@@ -3263,7 +3263,7 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly b PrivateIPC = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
-      readonly b PrivatePIDs = ...;
+      readonly s PrivatePIDs = '...';
```

Comment:
oha, can you just squash this fixup patch into the first commit. just mention in the commit msg that Daan did the original work.